resources/assets/js/clipboard.js

```diff
@@ -43,7 +43,8 @@ window.clipboard = () => {
             textArea.style.cssText =
                 "position:absolute;top:0;left:0;z-index:-9999;opacity:0;fontSize:12pt;";
 
-            document.body.append(textArea);
+            // @TODO: change to this.$root after apline 3 upgrade
```

Comment:
it seems to work for me on deployer in a modal with alpine v3, so this todo is obsolete? 🤔

Reply:
most browers will not get to it because they will use the `navigator.clipboard` bit instead but I will double check and adjust if required.

Reply:
locally it will run this part because of using http

Reply:
Both work but changing for $root because it's inline with alpine updates.